src/cli.py

```
@@ -300,6 +391,195 @@ def ssh_exec(host: str, username: str, key_file: Optional[str], command: str):
+            raise ValueError("'context.state' must be a JSON object")
+
+        context = AgentContext(
+            session_id=str(context_payload.get("session_id") or f"swarm_cli_{uuid4().hex[:10]}"),
+            user_id=context_payload.get("user_id"),
+            metadata=metadata,
+            state=dict(context_state),
+        )
+
+        resolved_tasks = _parse_task_inputs(list(tasks), tasks_json, payload)
+        if not resolved_tasks:
+            raise ValueError("No tasks provided. Use --task, --tasks-json, or payload task(s)")
```

Ruff (0.15.5)

[warning] Abstract `raise` to an inner function

(TRY301)